squid/docker-compose.yml

```diff
@@ -1,12 +1,43 @@
 version: "3"
 
 services:
+  node:
+    container_name: squid-node
```

Comment:
Can we name it, graphql-server (you can add node if you wish, to remember it's a node js, but that's easy to see)